### ISSUE TITLE
[Bug]: Tabs do not respond to `touchstart` events

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#11037]
### Package

@carbon/react

### Browser

Chrome

### Package version

v11

### Description

The tabs overflow scroll buttons do not respond to touch events, causing the buttons to not work in mobile environments.

https://user-images.githubusercontent.com/3360588/159559021-51588d17-6888-483b-8685-2f8b12b06983.mov

### CodeSandbox example

https://carbon-react-next.netlify.app/iframe.html?id=components-tabs--manual&args=&viewMode=story

### Steps to reproduce

Open the dev tools, toggle the device toolbar on, set it to `responsive`

Chrome now inputs only touch events to the dom, the buttons do not work

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
It looks like `usePressable` explicitly ignores touchstart due to text selection on iOS. In this case I think we could disable that, perhaps via a new parameter that turns off the behavior?

```js
// usage in component
usePressable(element, respondToTouchEventsBool);

// in usePressable.js
function onTouchStart(event) {
      // We prevent the default event on touchstart so that text selection is
      // disabled on iOS  Safari when interacting with a "pressable" element
      !respondToTouchEventsBool && event.preventDefault();
    }

    element.addEventListener('touchstart', onTouchStart);

**Comment by joshblack** [comment]
@tay1orjones I ended up just removing `touchstart` as I could not figure out a way around it 😞 Maybe someone else can take a crack at it to see if they can get it to work. I did add in a preventDefault for contextmenu's though as I saw that show up when using Chrome tools. Unfortunately, it seems like we'll have to ship with the Safari issue with text selection. I think this is an okay compromise as users will likely swipe the list instead (hopefully) but if we can figure out how to truly prevent this behavior that would be preferred.
